### PR TITLE
Feature/dynamic content

### PR DESCRIPTION
**Pull request by paulAcato**
Added the Dynamic content block.

The block has 2 render views, card and list. Card is the default view.

Screenshot 2022-06-20 at 11 59 40
Screenshot 2022-06-20 at 11 59 32
Screenshot 2022-06-20 at 11 59 23

The block has optional pagination. Therefor we used the existing pagination-component, for the links we used the existing link-component.

### CODE REVIEW

File: components/DynamicContent/src/stories/bem.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,846 @@
...
+  <Story name="Default">
+    <section class="denhaag-dynamic-content denhaag-dynamic-content--grid">
+      <div class="denhaag-dynamic-content__grid container">
+        <article class="denhaag-dynamic-content__card">
+          <img
+            class="denhaag-dynamic-content__card-image denhaag-image__image"
+            srcset="https://via.placeholder.com/340x190/90D40166 480w,
+                https://via.placeholder.com/510x285/90D40166 768w,
+                https://via.placeholder.com/680x380/90D40166 1290w"
+            src="https://via.placeholder.com/340x190/90D40166"
```
Comment (robbieatacato) [inline components/DynamicContent/src/stories/bem.stories.mdx:40]: This is the fallback image, when srcset is not supported.
So if this image is loaded in full width on desktop, it should be one of the higher resolutions.

File: components/DynamicContent/src/stories/bem.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,846 @@
...
+            alt=""
+            loading="lazy"
+          />
+          <div class="denhaag-dynamic-content__card-content">
+            <div class="denhaag-dynamic-content__card-meta">
```
Comment (robbieatacato) [inline components/DynamicContent/src/stories/bem.stories.mdx:45]: Should'nt 'denhaag-dynamic-content__card-meta' be more descriptive like 'denhaag-dynamic-content__card-caption'?

**Review by robbieatacato — CHANGES_REQUESTED** [review]

File: components/DynamicContent/src/index.scss [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    content: "";
+    left: 0;
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
+}
+
+.denhaag-dynamic-content__card-image {
+  @supports (aspect-ratio: 16 / 9) {
```
[the remark this answers is not in the document]
Comment (paulAcato) [reply components/DynamicContent/src/index.scss:68]: Old habit.

File: components/DynamicContent/src/stories/bem.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,846 @@
...
+
+## Stories
+
+### Default
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Default">
+    <section class="denhaag-dynamic-content denhaag-dynamic-content--grid">
+      <div class="denhaag-dynamic-content__grid container">
```
[the remark this answers is not in the document]
Comment (paulAcato) [reply components/DynamicContent/src/stories/bem.stories.mdx:33]: They're classes with other purposes.

I call it the grid, due we only apply grid-styles and no container styles (like max-width etc).
`.denhaag-dynamic-content__grid {
  display: grid;
  grid-template-columns: repeat(var(--denhaag-dynamic-content-grid-columns), 1fr);
  grid-column-gap: var(--denhaag-dynamic-content-grid-gap, 0);
  grid-row-gap: var(--denhaag-dynamic-content-row-gap);
}`

We're only placing the `.container` class so we don't need to add duplicate styles for the container.

An element may have multiple selectors, if the behavior is different and helps each other (this is how almost all CSS frameworks work).

**Pull request merged**